ComponentKit/Core/Scope/CKDetectComponentScopeCollisions.mm

```diff
@@ -31,9 +32,69 @@
 }
 #pragma clang diagnostic pop
 
-void CKDetectComponentScopeCollisions(const CKComponentLayout &layout)
+static CKComponent * findCollisionComponent(CKComponent *component,
+                                            id<NSObject> collisionScope,
+                                            NSMapTable<CKComponent *, CKComponent*> *componentsToParentComponents)
+{
+  for(CKComponent* componentKey in componentsToParentComponents){
```

Comment:
Nit: There should a space around the outside of the parentheses:

```
for (CKComponent *componentKey in componentsToParentComponents) {
```

Same feedback applies to conditionals.